data/podcasts.json

```diff
@@ -319,17 +319,6 @@
     "broadcast": "Tuesday",
     "logo": "https://mk0learntocodew6bl5f.kinstacdn.com/wp-content/themes/remixx/images/logo.svg"
   },
-    {
-    "name": "Insanely Chill",
-    "host": "Cody Ko",
-    "url": "http://insanelychillpodcast.com/",
-    "category": "comedy, interesting",
-    "description": "A podcast with guest with funny topics",
-    "languages": "English",
-    "updates": "bi weekly",
-    "broadcast": "fridays",
-    "logo": "http://insanelychillpodcast.com/images/podcast_logo_website.png"
```

Comment:
Thank you for your awesome contributions, I noticed you removed some of the listed podcast like the ones I have highlighted here, could you be nice to help fix the error.